gulpfile.babel.ts

```diff
@@ -29,7 +30,10 @@ const args = stdio.getopt({
     key: 'c',
     description: 'Copy dev build to Streak dev build folder',
   },
-});
+  examples: {
```

Comment:
I debated whether or not to include this flag. Examples take 1s to build, but I figured making them opt in was more explicit.